<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFlight</class>
 <widget class="QDialog" name="AddFlight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加飞机型号</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>飞机型号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>座位数目</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textFlightType">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textSeatNumber">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>97</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>97</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
